From c7871844503b3342b1a736c01246ecd13e3c2059 Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Fri, 23 Feb 2018 08:10:51 +0000
Subject: [PATCH 2/2] arm64: dts: sunxi: Orange Pi PC2 - disable RTC

Signed-off-by: Edmunt Pienkowsky <roed@onet.eu>

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts b/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
index 1010c1b22d2e..10f348e360d4 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
@@ -86,6 +86,10 @@ reg_usb0_vbus: usb0-vbus {
 	};
 };
 
+&rtc {
+	status = "disabled";
+};
+
 &codec {
 	allwinner,audio-routing =
 		"Line Out", "LINEOUT",
