From 823eee1dec0454a63e6b80fe3c7dad477d89003d Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Fri, 23 Feb 2018 08:10:51 +0000
Subject: [PATCH 2/2] arm64: dts: sunxi: Orange Pi PC2 - disable RTC

Signed-off-by: Edmunt Pienkowsky <roed@onet.eu>

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts b/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
index 3e0d5a9c096d..f27c78ffb5f5 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
@@ -124,6 +124,10 @@
 	};
 };
 
+&rtc {
+	status = "disabled";
+};
+
 &codec {
 	allwinner,audio-routing =
 		"Line Out", "LINEOUT",
-- 
2.20.1

